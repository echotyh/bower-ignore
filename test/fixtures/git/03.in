# Generated by bower-ignore, please DO NOT modify
# = Bower Ignore Begin

# = Bower Ignore End
# = Bower Ignore End
