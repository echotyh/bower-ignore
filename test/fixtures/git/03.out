# Generated by bower-ignore, please DO NOT modify
# = Bower Ignore Begin
bower_components/comp1/*
!bower_components/comp1/dist
bower_components/comp2/*
!bower_components/comp2/dist
# = Bower Ignore End
# = Bower Ignore End
